hsmd/hsmd.c

```diff
@@ -7,6 +7,7 @@
  * by request, response.
  */
 #include "config.h"
+#include "hsmd/hsmd_wiregen.h"
```

Comment:
This is not needed, my emacs try to be the smart guy here

Reply:
agreed, don't think it is needed, same above